### DISCUSSION TITLE
attechment support

### DISCUSSION THREAD

**Discussion by kshitijsoni** [koordinates/kart#578] [Google Soc / SoD]
Hi. I want to contribute to the attachment support project. Any starting challenge? @olsen232

**Comment by olsen232** [comment]
I've written some more about it here: https://github.com/koordinates/kart/issues/583
If I understand correctly, in the GSOC timeline we are currently at "Potential GSoC contributors discuss application ideas with mentoring organizations" so no need to jump into coding right away.

There's a label for issues called ["good first issue"](https://github.com/koordinates/kart/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) - there's only a couple of things in it right now unfortunately, and nothing that is truly straight forward, but we'll try and add to it. Anyone can try to take on such an issue to familiarise themselves with Kart a little bit
